R/rating.R

```diff
@@ -48,7 +62,7 @@ rating <- function(name, label = "", value = 0, max = 3, icon = "star",
 
 #' Update rating
 #'
-#' check \code{rating} to learn more.
+#' check \code{rating_input} to learn more.
```

Comment:
This isn't very informative :D sth like: `Change the value of a rating input on the client` ?